docs/benchmarking-workflow-corefx.md

```diff
@@ -240,6 +242,7 @@ namespace System
     public static class Console
     {
         public static void WriteHelloWorld() => WriteLine("Hello World!");
+        <!-- markdownlint-disable MD009 -->        
```

Comment:
nit: I am not sure about this line, GH shows it in the sample code and it looks a little weird:

![obraz](https://user-images.githubusercontent.com/6011991/54634508-a7a01180-4a82-11e9-826c-9435d4f2fc05.png)

Reply:
@billwert You need to change ```cs with ```csharp in the block comment.

Reply:
nah, the actual problem is that the XML comment was not rendering properly in the code block (which makes sense.) cs and csharp seem the same.

I'm just going to delete the blank line instead. :)